apollo-compiler/src/test/kotlin/com/apollographql/apollo3/compiler/PerfTest.kt

```diff
@@ -0,0 +1,102 @@
+package com.apollographql.apollo3.compiler
+
+import com.apollographql.apollo3.annotations.ApolloExperimental
+import com.apollographql.apollo3.compiler.TargetLanguage.KOTLIN_1_5
+import com.apollographql.apollo3.compiler.TestUtils.checkTestFixture
+import org.junit.Test
+import java.io.File
+import kotlin.time.ExperimentalTime
+import kotlin.time.measureTime
+
+@OptIn(ExperimentalTime::class, ApolloExperimental::class)
+class PerfTest {
```

Comment:
Could we do everything from `CodegenTest` ? `CodegenTest` writes a `measurements` file containing the codegen time IIRC so it might be enough?

Reply:
Yeah I really don't think we should push that test and indeed it should be one of the regular tests ran by `CodegenTest` (in fact it already does because where the files are located, and fails because I didn't generate the .expected 😅). But since it's fairly slow, and we may want to continue investigations, it's easier to have a dedicated test temporarily. If we want to merge this branch, let's remove or move it first.

Reply:
Would be nice to merge the other fixes quickly. My moving the test to another PR?

Reply:
👍  let's do it!